packages/playwright-core/src/server/injected/selectorGenerator.ts

```diff
@@ -237,7 +237,7 @@ function buildNoTextCandidates(injectedScript: InjectedScript, element: Element,
 
   const labels = getElementLabels(injectedScript._evaluator._cacheText, element);
   for (const label of labels) {
-    const labelText = label.full.trim();
```

Comment:
Is that expected that it is now normalized?

Reply:
Yep.